src/gui/widgets/dynarec_disassembly.cc

```diff
@@ -18,8 +18,11 @@
  ***************************************************************************/
 
 #include "gui/widgets/dynarec_disassembly.h"
-
+#ifdef _WIN32
```

Comment:
If this is needed it means the WIndows project is badly set up. Is `#include <capstone.h>` the canonical include path for it on Linux and MacOS?

Reply:
This is the correct include for Unix/Linux. It can break otherwise with the inability to find the files. So I added this to keep Windows builds working as well as Unix builds.